Create LLSEJSPlugin.md

Colton1070 opened the pull request:
Creating a very simple guide for novice developers to help them get started developing JS scripts for LiteLoaderBDS. This page could be expanded to a full guide showing common methods and techniques (like config files, error handling, etc.) - things that are generally JS specific knowledge but have a strong use case for LiteLoaderBDS Development. I do not know Chinese so I will be unable to translate this file. I welcome modifications but the general purpose of the file should remain intact. This guide takes inspiration from Discord.js's Guide which can be found here for reference: https://discordjs.guide/#before-you-begin

futrime commented on en/LLSEPluginDevelopment/ScriptAPI/LLSEJSPlugin.md:
```diff
@@ -0,0 +1,60 @@
+
+Before developing your first plugin, we will need to set up your developer environment (IDE). The choice of software you use to program is yours, but trusted and reliable software is recommended. [Atom](https://atom.io/) is a very simple program which can be used for this purpose.
+
+You will also need to set up a clean LiteLoaderBDS installation, details on how to install LiteLoaderBDS can be found [here](https://github.com/LiteLDev/LiteLoaderBDS#-install). This server will be used to test your plugin.
```
Here you should write the link to the documentation (https://docs.litebds.com/en/#/Usage).

futrime commented on en/LLSEPluginDevelopment/ScriptAPI/LLSEJSPlugin.md:
```diff
@@ -0,0 +1,60 @@
+
+## Prerequisites
+
+Before developing your first plugin, we will need to set up your developer environment (IDE). The choice of software you use to program is yours, but trusted and reliable software is recommended. [Atom](https://atom.io/) is a very simple program which can be used for this purpose.
```
You can also recommend VSCode, another widely used editor.

futrime commented on en/LLSEPluginDevelopment/ScriptAPI/LLSEJSPlugin.md:
```diff
@@ -0,0 +1,60 @@
+
+With your IDE in hand, and the server installation complete, you are ready to begin!
+
+## What do I do now?
```
Titles should be formatted. Refer to https://capitalizemytitle.com/

Futrime reviewed (changes requested):
You've done a good job! The docs are very close to perfect. Please modify some details based on my comments.

As for the Chinese translation problem, you needn't mind it. Just copy this file to the same place in /zh-Hans, I'm sure that someone will translate soon.

Colton1070 commented:
I had some issues navigating GitHub - I am still very new to repos. Hopefully the changes I made make sense. I have updated the page, moved it, and added it to the en sidebar. I noticed that the sidebar was linking parts of the docs:
- [📜 Using Javascript to Create Your First Plugin](/LLSEPluginDevelopment/LLSEJSPlugin.md)
Whereas in the LLSEJSPlugin.md, I used full links. Should I make another commit to change those links or is it ok?

Futrime reviewed (approved):
Good job.
